docs/en/integrations/data-ingestion/data-formats/csv-tsv.md

```diff
@@ -0,0 +1,389 @@
+---
+sidebar_label: CSV and TSV
+sidebar_position: 1
+slug: /en/integrations/data-formats/csv-tsv
+---
+
+# Working with CSV and TSV data in ClickHouse
+
+ClickHouse supports importing data from and exporting to CSV. Since CSV files can come with different format specifics, including header rows, custom delimiters, and escape symbols, ClickHouse provides formats and settings to address each case efficiently.
+
+
+## Importing data from a CSV file
+
+Before importing data, let’s create a table with a relevant structure:
+
+```sql
+CREATE TABLE sometable
+(
+    `path` String,
+    `month` Date,
+    `hits` UInt32
+)
+ENGINE = MergeTree
+ORDER BY tuple(month, path)
+```
+
+
+To import data from the [CSV file](assets/data_small.csv) to the `sometable` table, we can pipe our file directly to the clickhouse-client:
+
+
+```bash
+clickhouse-client -q "INSERT INTO sometable FORMAT CSV" < data_small.csv
+```
+
+Note that we use [FORMAT CSV](/docs/en/interfaces/formats.md/#csv) to let ClickHouse know we’re ingesting CSV formatted data. Alternatively, we can load data from a local file using the [FROM INFILE](/docs/en/sql-reference/statements/insert-into.md/#inserting-data-from-a-file) clause:
+
+
+```sql
+INSERT INTO sometable
+FROM INFILE 'data_small.csv'
+FORMAT CSV
+```
+
+Here, we use the `FORMAT CSV` clause so ClickHouse understands the file format. We can also load data directly from URLs using [url()](/docs/en/sql-reference/table-functions/url.md/) function or from S3 files using [s3()](/docs/en/sql-reference/table-functions/s3.md/) function.
+
+
+### CSV files with headers
```

Comment:
Actually after https://github.com/ClickHouse/ClickHouse/pull/44953 the header can be detected automatically in `CSV` format (So, actually `CSVWithNames/CSVWithNamesAndTypes` input formats are not needed anymore). I think we can add it to this doc woth examples.

Reply:
Agreed. That feature will be released in what version? So I follow and add docs once we're public with that.

Reply:
It's released in 23.1 yesterday

Reply:
ok, doing it

Reply:
PR: https://github.com/ClickHouse/clickhouse-docs/pull/711